packages/salesforcedx-sobjects-faux-generator/src/messages/localization.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as util from 'util';
```

Comment:
Decided to copy this from the utils module since it's the only meaningful piece of code that we are still using from that module. Doing this also helps us get rid of the deep link issues caused by using the utils module and sobject-faux module being used on a salesforcedx-vscode-* module.